docs/basic-docs/atomicdex-api-20-dev/get_locked_amount.md

```diff
@@ -0,0 +1,84 @@
+# get\_locked\_amount
+
+The `get_locked_amount` method returns the amount of a coin which is currently locked by a swap which is in progress. If the coin queried is not activated, a `NoSuchCoin` error will be returned.
+
+
+#### Arguments
+
+| Parameter | Type   | Description                                                          |
+| --------- | ------ | -------------------------------------------------------------------- |
+| coin      | string | The ticker of the coin you want to query.                            |
+
+
+#### Response
+
+| Parameter                    | Type            | Description                                                                                                    |
+| ---------------------------- | --------------- | -------------------------------------------------------------------------------------------------------------------- |
+| coin                         | string          | The ticker of the coin you queried.                                                                                  |
+| locked_amount                | object          | An object cointaining the locked amount in decimal, fraction and rational formats.                                   |
+| locked_amount.decimal        | numeric string  | The locked amount in [decimal format](https://www.mathsisfun.com/definitions/decimal.html).                          |
+| locked_amount.rational       | rational object | The locked amount in [rational format](../atomicdex-api-legacy/rational_number_note.md).                             |
+| locked_amount.fraction       | fraction object | The locked amount in [fraction format](https://www.mathsisfun.com/definitions/fraction.html).                        |
+| locked_amount.fraction.numer | numeric string  | The [numerator](https://www.mathsisfun.com/definitions/numerator.html) for the locked amount in fraction format.     |
+| locked_amount.fraction.denom | numeric string  | The [denominator](https://www.mathsisfun.com/definitions/denominator.html) for the locked amount in fraction format. |
+
+
+#### :pushpin: Examples
+
+#### Command
+
+```bash
+curl --url "http://127.0.0.1:7783" --data "{
+    \"userpass\": \"YOUR_PASS\",
+    \"mmrpc\": \"2.0\",
+    \"method\": \"get_locked_amount\",
+    \"params\": {
+        \"coin\": \"RICK\"
+    }
```

Comment:
Even if id is not mentioned in request/response fields descriptions, it's still worth having it in request example. Moreover, `"id": 42` is still in response examples.